src/applications/check-in/components/layout/tests/Footer.unit.spec.jsx

```diff
@@ -87,9 +74,22 @@ describe('check-in', () => {
           <Footer isPreCheckIn={false} />
         </CheckInProvider>,
       );
-      expect(component.getByTestId('travel-help-block')).to.exist;
+      expect(component.getByTestId('help-block')).to.exist;
+      expect(component.getByTestId('for-questions-about-filing')).to.exist;
     });
-    it('Renders TravelHelpBlock for complete page', () => {
+    it('Renders HelpBlock for complete page pre-check and travel section', () => {
```

Comment:
this is a little confusing to me. This is this a pre-check-in test or day-of? The Provider sets the app to pre-check-in but the footer says its not pre-check-in and then you are testing for day-of check-in things. I would refine this one a bit.